boards/arm/rcar_h3ulcb/rcar_h3ulcb_cr7.yaml

```diff
@@ -11,6 +11,7 @@ supported:
   - gpio
   - clock_control
   - uart
+  - pwm
```

Comment:
Should maybe set to unsupported waiting for #48834 to be merged ? Twister suite will report errors in this state !
What do you think @henrikbrixandersen, it depend on Zephyr test suite politic, maybe you are more aware than me ?

Reply:
Either that or exclude the board from the tests for now.

Reply:
Yes indeed, otherwise the twister test will failed. I removed it !
I did a push force again but everything else remains intact :)

Reply:
Thanks